lib/pure/os.nim

```diff
@@ -1210,7 +1210,7 @@ proc symlinkExists*(link: string): bool {.rtl, extern: "nos$1",
 
 
 when not defined(windows):
-  const maxSymlinkLen = 1024
+  const maxSymlinkLen = 4096
```

Comment:
Is this change required for decoupling `initOptParser` from `os`? If not, please make a separate PR for it.